project/SensibleSettings.scala

```diff
@@ -103,7 +103,6 @@ object Sensible {
 
   val scalaModulesVersion = "1.0.4"
   val akkaVersion = "2.3.14"
```

Comment:
Actually I chose 2.0.4 version of akka-http/streams for now, because I think it's the highest version that doesn't need akka 2.4.x.  Bumping akka to 2.4.x results for start with following error:
`[error] com.typesafe#config;1.2.1 (needed by [com.typesafe.slick#slick_2.11;3.1.1]) conflicts with com.typesafe#config;1.3.0 (needed by [com.typesafe.akka#akka-actor_2.11;2.4.3])`